### PR TITLE
add DOIs and source type tags up to August 11, 2022

### PR DESCRIPTION
**Pull request by miseminger**

### CODE REVIEW

File: data/RdRp_pharmaceutical_effectiveness.txt [hunk 1]
```diff
@@ -1,135 +1,135 @@
...
 # Report a 2.5x reduction in remdesivir EC50 in the presence of this mutation.
-# Szemiel et al (2021) https://doi.org/10.1371/journal.ppat.1009929
+# Szemiel et al (2021) http://doi.org/10.1371/journal.ppat.1009929 [Journal publication]
 E802D
 
 # Remdesivir failure with SARS-CoV-2 RNA-dependent RNA-polymerase mutation in a B-cell immunodeficient patient with protracted Covid-19
 # We report the occurrence of a mutation in the RdRP (D484Y) following failure of remdesivir in a 76-year-old woman with a post-rituximab 
 # B-cell immunodeficiency and persistent SARSCoV-2 viremia.
-# Martinot et al. (2020) https://doi.org/10.1093/cid/ciaa1474
+# Martinot et al. (2020) http://doi.org/10.1093/cid/ciaa1474 [Journal publication] [Journal publication]
```
Comment (nodrogluap) [inline data/RdRp_pharmaceutical_effectiveness.txt]: Duplicate type tag

**Review by nodrogluap — APPROVED** [review]
One dupe publication type tag.

**Comment by miseminger** [comment]
Source types corrected now (had initially put types for preprints that had been published after curation)

**Comment by miseminger** [comment]
Source types (for now--can be changed) are:

['Journal publication',
 'Preprint (MedRxiv)',
 'Preprint (other)',
 'Preprint (BioRxiv)',
 'Grey literature']

**Pull request merged**